### PR TITLE
[3] v2.1.0 and update changelog

### PR DESCRIPTION
**Pull request by trel**
confirming this builds as py2/3 universal as expected.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,39 @@
 Changelog
 =========
 
+v2.1.0 (2024-08-13)
+-------------------
+- [#3] v2.1.0 and update changelog [Terrell Russell]
+- [#3] allow genquery2_test to work under all Python2/3 versions [Daniel Moore]
+- [#3] call assertRaisesRegex[p] with/without final 'p' (depends on Python 2 or 3) [Daniel Moore]
```
[the remark this answers is not in the document]
Comment (trel) [reply CHANGELOG.rst:8]: the difference between `assertRaisesRegex` and `assertRaisesRegexp`

[the remark this answers is not in the document]
Comment (trel) [reply CHANGELOG.rst:8]: https://github.com/irods/python-irodsclient/commit/9b3c7b32d0c0fdcebe2f5c8dbf2fe71d6c4d04c0

[the remark this answers is not in the document]
Comment (korydraughn) [reply CHANGELOG.rst:8]: I see now. Thanks.

**Comment by trel** [comment]
Installs and imports locally as expected - I think it's ready.

Will # upon command.

**Comment by trel** [comment]
#'d

**Pull request merged**